Bottom: 87e7689cda54052627876bc13ddc03e152218139
Top:    e933c3c2d604de8a476e6e31608588ee6f2dd67a
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-16 14:10:08 -0400

Refresh of stac9766

---

diff --git a/sound/soc/codecs/stac9766.c b/sound/soc/codecs/stac9766.c
index 75a3932..2f4f2f8 100644
--- a/sound/soc/codecs/stac9766.c
+++ b/sound/soc/codecs/stac9766.c
@@ -412,7 +412,6 @@ int stac9766_reset(struct snd_soc_codec *codec, int try_warm)
 		return -EIO;
 	return 0;
 }
-EXPORT_SYMBOL_GPL(stac9766_reset);
 
 static int stac9766_codec_probe(struct platform_device *pdev)
 {
